_sections/home-english.md

```diff
@@ -748,27 +748,32 @@ An example **settings** worksheet is below:
 
 The column headings in this example **settings** worksheet do the following:
 
-* **form_title**: The title of the form that is shown to users.  The form title is pulled from **form_id** if **form_title** is blank or missing.
-* **form_id**: The name used to identify the form submission.  The form id is pulled from the XLS file name if **form_id** is blank or missing.
-* **public_key**: For encrypted forms, this is where the public key is copied and pasted.
-* **submission_url**: For encrypted forms, this url specifies the server where finalized forms are submitted to.
+* **form_title**: The title of the form that is shown to users. The form title is pulled from **form_id** if **form_title** is blank or missing.
+* **form_id**: The name used to uniquely identify the form on the server. The form id is pulled from the XLS file name if **form_id** is blank or missing.
+* **public_key**: For encryption-enabled forms, this is where the public key is copied and pasted.
+* **submission_url**: This url can be used to override the default server where finalized records are submitted to.
 * **default_language**: In localized forms, this sets which language should be used as the default.
 * **version**: String of up to 10 numbers that describes this revision. Revised form definitions must have alphabetically greater versions than previous ones. A common convention is to use strings of the form 'yyyymmddrr'. For example, 2017021501 is the 1st revision from Feb 15th, 2017.
 
 ### Encrypted forms
 
-Encrypted forms provide a mechanism to keep your data private using http for communication. Form submissions sent to the Aggregate server are encrypted and completely inaccessible to anyone not possessing the private key.
+Encryption-enabled forms provide a mechanism to keep finalized records private at all times. This includes the time records are stored on the device and server as well as during transport, even when http is used for communication. Encrypted records including their uploaded files, such as photos, are completely inaccessible to anyone not possessing the private key.
```

Comment:
I'm wondering whether this is a little bit ambiguous? "finalized records" is specified in the first sentence but then the second sentence could be taken to imply records are encrypted as they're being filled out which is not the case across clients (could get very slow). To be even more explicit, maybe something like "This includes the time after finalization that records are..." Or alternately "Note that un-finalized records are not (may not be?) stored encrypted."

Reply:
yes, very good